packages/core/src/native_core/index.ts

```diff
@@ -21,6 +22,8 @@ try {
     isBound: true,
   };
 } catch (e) {
+  logDebug("Failed to bind native core, instruments will not work.");
```

Comment:
When under the runner, we should always put the message, and fail when working with the runner

Reply:
I've kept the actual message as debug, and added an error throw when calling setupCore with no native core, which should not happen if users use the library without the runner.